scripts/check-server-online.py

```diff
@@ -0,0 +1,49 @@
+"""
+Block until server responds.
+
+This script is used by CI to ensure tests are not run while server is starting.
+"""
+
+import argparse
+import time
+
+import urllib3
+
+from camayoc import api
+
+urllib3.disable_warnings()
+
+
+class ServerUnavailableException(Exception):
+    pass
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog="check-server-online", description="Block until server responds"
+    )
+    parser.add_argument(
+        "--timeout", default=30, type=int, help="How long to wait before giving up (seconds)"
+    )
+    args = parser.parse_args()
+
+    start_time = time.monotonic()
+
```

Comment:
Providing a safe default in case the looping `api_client.get` never raises an exception but also never gets a response with 200 status:
```suggestion

last_exception = None
```